doc/lss.rst

```diff
@@ -46,6 +46,8 @@ Or, you can call this method with 4 IDs if you want to switch only one slave::
 Or, you can run fastscan procedure ::
 
     ret_bool, lss_id_list = network.lss.fast_scan()
+    
+**NOTE**: Before you run fastscan, you have to make sure the devices are in non-configured or unconfigured state.    
```

Comment:
Could use [make it stand out more](https://sublime-and-sphinx-guide.readthedocs.io/en/latest/notes_warnings.html).

```suggestion
.. note::
   Before you run fastscan, you have to make sure the devices are in non-configured or unconfigured state.
```